plugin/cscope_dynamic.vim

```diff
@@ -208,6 +208,10 @@ function! s:dbUpdate()
             let cmd .= " | grep -v -f".s:small_file.".files "
         endif
 
+        " Trick to resolve links with relative paths
+        "
+        let cmd .= "| xargs readlink -f "
```

Comment:
This should be guarded with
   if s:resolve_links

I'm also not sure about this translating into absolute paths as it's changing behavior. You can retain the relative paths by using this command instead
   xargs realpath --relative-to=$(pwd)

If you desire absolute paths, then I think it would be better to add a new config option; g:cscopedb_absolute_paths.